src/main/java/org/konstructs/grass/GrassActor.java

```diff
@@ -109,13 +107,13 @@ public void onEventBlockUpdated(EventBlockUpdated blockEvent) {
     public void onBoxQueryResult(BoxQueryResult result) {
 
         // Expect a 3x4x3 box
-        if (result.result().data().size() != 3*4*3) {
-            System.out.println("onBoxQueryResult: Error: Got a box with size " + result.result().data().size());
+        if (result.getBlocks().length != 3*4*3) {
```

Comment:
The code was added there as a test when learning to use the boxQuery ... just left it there as a sanity check.